.cursor/Dockerfile

```
@@ -0,0 +1,13 @@
+# Cloud Agent / Dev Container base image for Operation Prometheus.
+# Cursor checks out the workspace separately — do not COPY the project here.
+FROM python:3.12-bookworm
+
+RUN apt-get update \
```

Hadolint (2.15.1)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)